### DISCUSSION TITLE
Select2 in list

### DISCUSSION THREAD

**Discussion by soluzione-software** [Laravel-Backpack/community-forum#636] [Q&A (Help)] [answered]
I want to add a select2 field in list's bottom, for example i coud add a button to bottom of my list with:
$this->crud->addButton('bottom', 'bulk_clone', 'view', 'bulk_clone', 'beginning');
but i want to add (to the left of button) a select2 fields as I do in CRUD, for sample:
CRUD::field([   // select_from_array
    'name'        => 'template',
    'label'       => "Template",
    'type'        => 'select_from_array',
    'options'     => ['one' => 'One', 'two' => 'Two'],
    'allows_null' => false,
    'default'     => 'one',
    // 'allows_multiple' => true, // OPTIONAL; needs you to cast this to array in your model;
]);

how can i do this?

**Comment by pxpm** [comment]
Hey @soluzione-software .

I'd suggest you create a custom button https://backpackforlaravel.com/docs/6.x/crud-buttons#creating-a-custom-button-1

In that view you can have the select, the button and the JS (if needed) to control the select/button behavior. 

There is no stock option built in Backpack for that, sorry.

Cheers 🙏

**Comment by soluzione-software** [comment, marked as answer]
ok I solve.
simply in view:
$field=[   // select_from_array
    'name'        => 'template',
    'label'       => "Template",
    'type'        => 'select_from_array',
    'options'     => ['one' => 'One', 'two' => 'Two'],
    'allows_null' => false,
    'default'     => 'one',
    // 'allows_multiple' => true, // OPTIONAL; needs you to cast this to array in your model;
];
@include('crud::fields.'.$field['type'], array('field' => $field))

**Reply by pxpm** [reply]
Thanks for sharing your solution 🙏
